lib/hexpm_web/templates/docs/usage.html.md

```diff
@@ -43,7 +43,7 @@ end
 
 <dl class="dl-horizontal">
   <dt><code>:hex</code></dt>
-  <dd>The name of the package. Defaults to the dependency application name.</dd>
+  <dd>The name of the package on Hex. Defaults to the dependency application name. Use this option to override if the dependency sets different <code>name</code> (mix) or <code>pkg_name</code> (rebar3).</dd>
```

Comment:
I think this is a little clearer for people that do not know the configuration options when publishing dependencies.

 ```suggestion
  <dd>The name of the package on Hex. Defaults to the dependency application name. Use this option if the dependency has different OTP application and package names.</dd>
```

Reply:
In wouldn't make searching much easier if the relevant rebar3/mix options are not referenced here

Reply:
What search are you referring to? I don't think we should expect consumers of packages to know or learn options for publishing since <1% of users actually publish packages. Mentioning the `name` / `pkg_name` is only useful for users that know of those publish options, as the text is written in the PR it would be easy ti think those options are somehow part of the dependency declaration.

Reply:
As a consumer of rebar3 package that sets pkg_name (I can see dat in .app.src) I'd like to be able to find the missing option by a simple google search. Try searching with `mix dependency pkg_name` or similar queries and see if it is easy to find.

Reply:
What do you think about doing this change on the publish documentation in that case? Since that's where `name` and `pkg_name` is documented. I don't think consumers of packages should have to learn about options they are not actually using.

Reply:
It would certainly be helpful. Obviously the best solution would be to make the options aligned. Now there are 4 variants (name, pkg_name, pkg, hex)

Reply:
Do you have any ideas on how to align them? For me it feels difficult since it's two different build tools and the options are used in different contexts.

Reply:
Even 3 build tools if you include erlang.mk. Of course changing them would be breaking and require some transitory period.